CHANGELOG.md

```diff
@@ -9,6 +17,7 @@
 ### Breaking Changes
 
 - Conditions that require to specify a pet now correctly check that. `[ hp > 1 ]` was never valid and thus never true, but did not provoke an error to hint users to change it to `[ self.hp > 1 ]`. This change only breaks scripts that are currently silently broken already.
+- Plugins that use extra data in import/export strings are now forced to import the data without user choice. The plugin API has changed from `Plugin:OnImport(extra)` to `Plugin:OnImport(script, extra)` to allow for mapping between script and extra data.
```

Comment:
Is this too technical for a changelog entry? To be honest, I am not too sure what it means myself :P

Reply:
This addon provides a feature for third party developers to hook into. It is information for them, which needs to be technical. 

No, I don't know whether there are any such developers.